src/plugins/intel_npu/src/plugin/src/backends.cpp

```diff
@@ -163,6 +163,11 @@ bool NPUBackends::isBatchingSupported() const {
     return false;
 }
 
+bool NPUBackends::isCompilerFallbackRequired() const {
+    static const uint32_t lastCompatibleVersion = 2196;
```

Comment:
I think this method will be updated and, in the end removed in the future. This check is added in order to prevent CID fallback for some drivers (that don't have graph extension 1.6) that could use CIP instead. However, CIP is continuously being developed and I don't expect for the compatibility between older drivers and CIP to persist. When the last driver that doesn't contain graph extension 1.6 will be 'not supported' by CIP, we will be able to remove this check altogether.

Reply:
Does it work for both linux and windows drivers?

Reply:
I've updated this check to work for both